webpack.dev.js

```diff
@@ -0,0 +1,42 @@
+import { resolve as _resolve } from 'path';
+
+import pkg from 'webpack';
+import merge from 'webpack-merge';
+
+import common from './webpack.common';
```

Comment:
running into an issue with importing `webpack.common`
<img width="900" alt="image" src="https://github.com/readmeio/mdx/assets/43232318/55aa8c98-ecce-48bb-b637-b11e87809e4a">

Reply:
I _think_ for ES modules, the extension is required:

```suggestion
import common from './webpack.common.js';
```

Reply:
ah gotcha